L1Trigger/L1TTrackMatch/plugins/L1TkHTMissEmulatorProducer.cc

```diff
@@ -138,6 +138,8 @@ void L1TkHTMissEmulatorProducer::produce(edm::Event& iEvent, const edm::EventSet
     //float tmp_jet_et_ = jetIter->pt();  // FIXME Get Et from the emulated jets
     float tmp_jet_pt_ = jetIter->pt();
 
+    int IsDisplaced = jetIter->dispflag();
```

Comment:
Does this flag matter for downstream usage of the extended jets? (and do we have it available in the collection?)

Reply:
Not sure I understand the question. The displaced jet produces code this flag to identify a jet as displaced. This is already on jet output and can be used downstream. Here we  calculate the HT. The extendedHT is already in cmsww. The issue is that as it is right now, we calculate it from all jets (NOT the displaced only), which is wrong.